Makefile.js

```diff
@@ -1063,6 +1064,43 @@ target.checkGitCommit = function() {
 };
 
 /**
+ * Downloads a repository which has many js files to test performance with multi files.
+ * Here, it's eslint@1.10.3 (450 files)
+ * @param {function} cb - A callback function.
+ * @returns {void}
+ */
+function downloadMultifilesTestTarget(cb) {
```

Comment:
Can this checkout be done to a temp directory instead? Would be nice to keep it outside the project to avoid any potential issues.